platforms/cpuinfo.go

```diff
@@ -26,6 +27,7 @@ import (
 
 	"github.com/containerd/containerd/errdefs"
 	"github.com/containerd/containerd/log"
+	"golang.org/x/sys/unix"
```

Comment:
wouldn't this be *inx platform specific?

Reply:
Yes, it's only for Linux.  
I will move this func to a *._linux.go file.